api/server.go

```diff
@@ -445,6 +455,7 @@ func (s *server) txpoolBroadcastHandler(jc jape.Context) {
 			return
 		}
 	}
+	resp.Basis = tbr.Basis
```

Comment:
Doesn't this make the basis from line `405` redundant since it is always overwritten?